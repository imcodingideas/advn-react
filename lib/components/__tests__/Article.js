import React from 'react';
import Article from '../Article';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const article = {
    title: 'Test article',
    date: 'Test date',
    body: 'Test body',
    author: {
      firstName: 'Joseph',
      lastName: 'Chambers',
      website: 'test.com'
    }
  };
  const tree = renderer.create(
    <Article article={article} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});