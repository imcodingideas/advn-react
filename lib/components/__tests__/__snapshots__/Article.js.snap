// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    Test article
  </div>
  <div>
    Test date
  </div>
  <div>
    <div>
      Chambers, Joseph
    </div>
    <div>
      test.com
    </div>
  </div>
  <div>
    Test body
  </div>
</div>
`;
